index.html

```diff
@@ -21,7 +21,7 @@
            </nav>
       </header>
             <section id="Features">
-               <h>this features sections </h>
+               <h2>this features sections </h2>
             </section>
             <section id="How-It-Works">
               <h2>How It Works sections </h2> 
```

Comment:
All ways try to use one h1 in your code. This will allow the google search engine to find your site faster.